src/cmd/run.js

```diff
@@ -322,6 +322,8 @@ export class ExtensionRunner {
 
   run(profile: FirefoxProfile): Promise<FirefoxProcess> {
     const {firefoxApp, firefox} = this;
-    return firefoxApp.run(profile, {firefoxBinary: firefox});
+    return firefoxApp.run(
+        profile, {firefoxBinary: firefox, binaryArgs: ['-jsconsole']}
```

Comment:
@kumar303 When I debug this `log.debug(jsconsole)`, I am getting it as undefined.
So this line is not getting executed 
```
if (jsconsole) {
  binaryArgs.push('-jsconsole');
}
```
Do I need to define `jsconsole` anywhere else other than `ExtensionRunner.run()`